api/routers/users-model.js

```diff
@@ -30,3 +31,9 @@ function findById(id) {
         .where({ id })
         .first();
 }
+
+function findByUsername(username) {
```

Comment:
This function isn't working how I want... I need to research more to understand why

Reply:
that looks like it should query users by username. I think the issue is in the pulling username from params.